src/styles/card.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+  
+.card__price-value,
+.card__price-label,
+.card__title-label,
+.card_rate-label,
+.card__title {
+  margin: 0;
+  cursor: default;
+}
+  
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 199px;
+  align-items: center;
+  margin: 0 16px;
+}
```

Comment:
You should add paddings in this bloock, instead of adding them to each element

Reply:
done, thanks